### PR TITLE
Make confusable character warning less jarring

### PR DESCRIPTION
**Pull request by n0toose**
This commit assumes that the warning can be made more discreet
so as to make it less annoying for the people that do not actually
need the warning, without necessarily increasing the risk for those
that do need it.

This doesn't fix the underlying problem of the warning being shown
in certain cases that, say, a certain kind of whitespace character
like 0x1E could be absolutely justifiable from a technical
perspective.

### CODE REVIEW

**Comment by n0toose** [comment]
No `warning`'s and `error`'s are used anymore, as it is assumed that the large header is enough to attract the attention of the user.

I considered two cases:
- use `warning`'s in both cases
- only use `warning`'s in one of the two cases (but I wasn't too sure about that)

**Comment by delvh** [comment]
Any screenshot?

**Comment by n0toose** [comment]
> Any screenshot?

Sure, give me a second.

**Comment by n0toose** [comment]
Full disclosure that I should've mentioned earlier: Considering that the change is minor and does not contain any logic changes or anything that could otherwise prevent the site from rendering correctly, I recreated the change again using "Inspect Element" (removed the tag, changed the text content) on Codeberg to save time.

I didn't see a reason to do so, but I can test this thoroughly if necessary.

image

image

**Review by wxiaoguang — APPROVED** [review]

File: options/locale/locale_en-US.ini [hunk 1]
```diff
@@ -1131,10 +1131,10 @@ file_view_rendered = View Rendered
 file_view_raw = View Raw
 file_permalink = Permalink
 file_too_large = The file is too large to be shown.
-invisible_runes_header = `This file contains invisible Unicode characters!`
-invisible_runes_description = `This file contains invisible Unicode characters that may be processed differently from what appears below. If your use case is intentional and legitimate, you can safely ignore this warning. Use the Escape button to reveal hidden characters.`
-ambiguous_runes_header = `This file contains ambiguous Unicode characters!`
-ambiguous_runes_description = `This file contains ambiguous Unicode characters that may be confused with others in your current locale. If your use case is intentional and legitimate, you can safely ignore this warning. Use the Escape button to highlight these characters.`
+invisible_runes_header = `This file contains invisible Unicode characters`
+invisible_runes_description = `This file contains different kinds of invisible Unicode characters that are indistinguishable to humans, but may be processed differently by a computer. If you think that this is intentional, you can safely ignore this warning. Use the Escape button to reveal them.`
```
Comment (delvh) [inline options/locale/locale_en-US.ini]:
```suggestion
invisible_runes_description = `This file contains invisible Unicode characters that are indistinguishable to humans but may be processed differently by a computer. If you think that this is intentional, you can safely ignore this warning. Use the Escape button to reveal them.`
```

File: options/locale/locale_en-US.ini [hunk 2]
```diff
@@ -1131,10 +1131,10 @@ file_view_rendered = View Rendered
...
+ambiguous_runes_header = `This file contains ambiguous Unicode characters`
+ambiguous_runes_description = `This file contains ambiguous Unicode characters that could possibly be confused with other characters. If you think that this is intentional, you can safely ignore this warning. Use the Escape button to reveal them.`
```
Comment (delvh) [inline options/locale/locale_en-US.ini]:
```suggestion
ambiguous_runes_description = `This file contains Unicode characters that might be confused with other characters. If you think that this is intentional, you can safely ignore this warning. Use the Escape button to reveal them.`
```

**Comment by techknowlogick** [comment]
Thanks for this PR. I'm hesitant to approve with the warning removed, as this is in place to remedy a security concern and otherwise the security reporters had wanted to request a CVE without the warning.

**Comment by denyskon** [comment]
I think using warning in both cases would be the best compromise.

**Comment by n0toose** [comment]
> I think using warning in both cases would be the best compromise.

I addressed this.

**Review by denyskon — APPROVED** [review]

**Comment by denyskon** [comment]
@techknowlogick Do you want to block or is this compromise acceptable for you?

**Review by lunny — APPROVED** [review]

**Pull request merged**